### DISCUSSION TITLE
[Request] add secondary ethernet port

### DISCUSSION THREAD

**Discussion by asyba** [RaspAP/raspap-webgui#959] [Ideas]
Can be implemente a feature that apart of connecting to the AP with the WIFI created by raspAP we could also connect by cable? adding an external usb3-ethernet adapter.

**Comment by billz** [comment]
The Insiders Edition of RaspAP has support for multiple classes of attached network devices.
Info here: https://docs.raspap.com/net-devices/

**Reply by asyba** [reply]
I see is only available for insiders.

**Reply by billz** [reply]
It will be merged into the public repo when the funding target is met (Insiders sustains this project)
